tests/settings/ConfigurationGatheringTest.py

```diff
@@ -14,7 +14,11 @@
 from coalib.output.printers.LogPrinter import LogPrinter
 from coala_utils.string_processing import escape
 from coalib.settings.ConfigurationGathering import (
-    find_user_config, gather_configuration, load_configuration)
+    find_user_config,
+    gather_configuration,
+    get_filtered_bears,
+    load_configuration,
+    )
```

Comment:
rather like this, as this is our current style :)
```

```
If you dislike this, feel free to file an issue. This style is definitely worth thinking about if you believe that we should have this (because of VCS reasons) :)